src/shared/stores/index.js

```diff
@@ -87,4 +87,9 @@ export default types
       window.document.cookie = 'wppwaClassicVersion=true;path=/';
       window.location.reload(true);
     },
+    addComments(type, id) {
```

Comment:
Is this a fix for a different issue?

Reply:
Yes, this fix is being done in three different branches 😢 because it was throwing an error when loading a post, but it shouldn't give as any trouble in the merge, as the fix is identical every time.